executor/slow_query.go

```diff
@@ -148,10 +167,24 @@ func (e *slowQueryRetriever) getNextFile() *os.File {
 	return file
 }
 
+func (e *slowQueryRetriever) getPreviousFile() *os.File {
+	fileIdx := e.fileIdx
+	fileIdx = fileIdx - 2
```

Comment:
Why it is `fileIdx - 2`, not  `fileIdx - 1`?

Reply:
Because `getNextFile` will set the index to the next file which should be read instead of the current reading file.